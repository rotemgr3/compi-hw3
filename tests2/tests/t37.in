//piazza 96 - should be ok +
int foo() {
	while(true) {
		{
		return 5;
		}
		break;
	}
}
void main() {int booga = 5;}
 
