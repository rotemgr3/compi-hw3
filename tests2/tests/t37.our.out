---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
booga INT 0
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
foo ()->INT 0
main ()->VOID 0
